src/components/experiment/ExperimentVisitsTable.tsx

```diff
@@ -0,0 +1,119 @@
+import MaterialTable from '@material-table/core';
+import { makeStyles } from '@mui/styles';
+import { DateTime } from 'luxon';
+import React from 'react';
+
+import { GetScheduledEventsCoreQuery, TrainingStatus } from 'generated/sdk';
+import { tableIcons } from 'utils/materialIcons';
+import { getFullUserName } from 'utils/user';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    '& tr:last-child td': {
+      border: 'none',
+    },
+    '& .MuiPaper-root': {
+      padding: '0 40px',
+      backgroundColor: '#fafafa',
+    },
+  },
+  noVisit: {
+    textAlign: 'center',
+    padding: '20px',
+  },
+}));
+
+type RowType = NonNullable<
+  GetScheduledEventsCoreQuery['scheduledEventsCore'][0]['visit']
+>['registrations'][0];
+
+const getHumanReadableStatus = (rowData: RowType) => {
+  switch (rowData.trainingStatus) {
+    case TrainingStatus.ACTIVE:
+      return `Valid until ${DateTime.fromISO(
+        rowData.trainingExpiryDate
+      ).toFormat('dd-MM-yyyy HH:mm')}`;
+    case TrainingStatus.EXPIRED:
+      return `Expired ${DateTime.fromISO(rowData.trainingExpiryDate).toFormat(
+        'dd-MM-yyyy HH:mm'
+      )}`;
+    case TrainingStatus.NONE:
+      return 'Not started';
+    default:
+      return 'Unknown';
+  }
+};
+
+type ScheduledEvent = GetScheduledEventsCoreQuery['scheduledEventsCore'][0];
+
+interface ScheduledEventDetailsTableProps {
+  scheduledEvent: ScheduledEvent;
+}
+
+const formatDate = (date: Date | null) =>
+  date ? DateTime.fromJSDate(date).toFormat('yyyy-MM-dd') : 'N/A';
+
+function ExperimentVisitsTable({
+  scheduledEvent,
+}: ScheduledEventDetailsTableProps) {
+  const classes = useStyles();
+
+  const columns = [
+    {
+      title: 'Visitor name',
+      render: (rowData: RowType) => getFullUserName(rowData.user),
+    },
+    {
+      title: 'Teamleader',
+      render: (rowData: RowType) =>
+        rowData.userId === scheduledEvent.visit?.teamLead.id ? 'Yes' : 'No',
+    },
+    {
+      title: 'Starts at',
+      field: 'rowData.startsAt',
+      render: (rowData: RowType) =>
+        rowData.isRegistrationSubmitted
+          ? formatDate(rowData.startsAt)
```

Comment:
Here as well as mentioned above should follow this example: https://github.com/UserOfficeProject/user-office-frontend/blob/develop/src/components/call/CallsTable.tsx#L218